futures-util/src/compat/compat01as03.rs

```diff
@@ -413,6 +414,7 @@ mod io {
         ///
         /// ```
         /// #![feature(async_await, impl_trait_in_bindings)]
+        /// # #![allow(incomplete_features)]
```

Comment:
I don't see where this example uses the `impl_trait_in_bindings` feature

Reply:
Oops, I missed that. Removed